main.cpp

```diff
@@ -16,6 +16,23 @@ void PrintBoard()
         cout << endl;
     }
 }
+int Win(char board [SIZE][SIZE], char symbol) {
```

Comment:
лучше в таком случае возвращать `bool`
```suggestion
bool Win(char board [SIZE][SIZE], char symbol) {
```